CsCore/xUnitTests/xUnitTests.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
-	<LangVersion>8.0</LangVersion>
+	<LangVersion>8</LangVersion>
```

Comment:
Can you revert this change since its unrelated to the topic of the PR? You can manually edit the file with a text editor

Reply:
Or actually never mind, if your IDE did this change automatically lets keep it at this and not revert it, otherwise every other PR other people would create would also contain this change. So please just add a comment here that this was done by your IDE automatically